go/vt/vttest/local_cluster.go

```diff
@@ -497,9 +499,29 @@ func (db *LocalCluster) loadSchema(shouldRunDatabaseMigrations bool) error {
 	return nil
 }
 
+func (db *LocalCluster) createVTSchema() error {
```

Comment:
Note: to help understand the reviewer, this needs to be added explicitly in local_cluster because vttest doesn't follow the same code flow as vitess does.

Reply:
Added comments